src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -233,7 +233,7 @@
   <data name="CannotOpenFile" xml:space="preserve">
```

Comment:
@Benglin yes the changes are identical, although i ddnt overwrite Resources.en-US.resx. I just made sure the changes were the same. Is it recommended to always overwrite Resources.en-US.resx if changes are made to Resources.resx